integration-test/src/main/java/com/sequenceiq/it/cloudbreak/ImageCatalogTests.java

```diff
@@ -0,0 +1,209 @@
+    }
+
+    // BUG-97072
+    @Test(expectedExceptions = BadRequestException.class, enabled = false)
+    public void testCreateInvalidImageCatalogUrl() throws Exception {
+        given(CloudbreakClient.isCreated());
+        given(ImageCatalog.request()
+                .withName(VALID_IMAGECATALOG_NAME + "-url")
+                .withUrl("https://" + INVALID_IMAGECATALOG_URL), "an imagecatalog request with invalid url"
+        );
+        when(ImageCatalog.post(), "post the imagecatalog request");
+        then(ImageCatalog.assertThis(
+                (imageCatalog, t) -> {
+                    Assert.assertNotEquals(imageCatalog.getResponse().getName(), VALID_IMAGECATALOG_NAME + "-url");
+                }),  "check imagecatalog is not created with invalid url");
+    }
+
+    @Test(expectedExceptions = BadRequestException.class)
+    public void testCreateInvalidImageCatalogUrlNoProtocol() throws Exception {
+        given(CloudbreakClient.isCreated());
+        given(ImageCatalog.request()
+                .withName(VALID_IMAGECATALOG_NAME + "-url-np")
+                .withUrl(INVALID_IMAGECATALOG_URL), "an imagecatalog request with url no protocol"
+        );
+        when(ImageCatalog.post(), "post the imagecatalog request");
+        then(ImageCatalog.assertThis(
+                (imageCatalog, t) -> {
+                    Assert.assertNotEquals(imageCatalog.getResponse().getName(), VALID_IMAGECATALOG_NAME + "-url-np");
+                }),  "check imagecatalog is not created with url no protocol");
+    }
+
+    @Test(expectedExceptions = BadRequestException.class)
+    public void testCreateDeleteValidCreateAgain() throws Exception {
+        given(CloudbreakClient.isCreated());
+        given(ImageCatalog.isCreatedDeleted()
+                .withName(VALID_IMAGECATALOG_NAME + "-delete-create")
+                .withUrl(VALID_IMAGECATALOG_URL), "an imagecatalog request then delete"
+        );
+        given(ImageCatalog.request()
+                .withName(VALID_IMAGECATALOG_NAME + "-delete-create")
+                .withUrl(INVALID_IMAGECATALOG_URL), "same imagecatalog request again"
+        );
+        when(ImageCatalog.post(), "post the imagecatalog request");
+        then(ImageCatalog.assertThis(
+                (imageCatalog, t) -> {
+                    Assert.assertEquals(imageCatalog.getResponse().getName(), VALID_IMAGECATALOG_NAME + "-delete-create");
+                }),  "check imagecatalog is created when name was used and deleted before");
+    }
+
+    @Test(expectedExceptions = BadRequestException.class)
+    public void testDeleteCbDefaultImageCatalog() throws Exception {
+        given(CloudbreakClient.isCreated());
+        given(ImageCatalog.request()
+                .withName(DEFAULT_IMAGECATALOG_NAME)
+        );
+        when(ImageCatalog.delete(), "try to delete cb default image catalog");
+    }
+
+    @Test
+    public void testSetNewDeleteDefaultImageCatalog() throws Exception {
```

Comment:
I am not really getting this name, could be two test with "testSetDefaultImageCatalog" and "testDeleteDefaultImageCatalog"